source/bases/Win32Service.c

```diff
@@ -468,45 +500,70 @@ static int Service_Install(wchar_t *name, wchar_t *configFileName)
 //-----------------------------------------------------------------------------
 static int Service_Uninstall(wchar_t *name)
```

Comment:
I noticed that reference counts in `Service_Install` are not handled, but they are handled in `Service_Uninstall`. Was `Service_Install` left out intentionally? In any case, my main concern is the reference counts in  `Service_SetupPython` and `Service_Run` are addressed (which they are), since any memory leak as part of install or uninstall will be very short lived.